plugins/inputs/gnmi/gnmi_test.go

```diff
@@ -423,11 +423,11 @@ func TestSubscribeResponseError(t *testing.T) {
 	var mc uint32 = 7
 	ml := &MockLogger{}
 	plugin := &GNMI{Log: ml}
-	// TODO: FIX SA1019: gnmi.Error is deprecated: Do not use.
-	errorResponse := &gnmi.SubscribeResponse_Error{Error: &gnmi.Error{Message: me, Code: mc}}
-	plugin.handleSubscribeResponse("127.0.0.1:0", &gnmi.SubscribeResponse{Response: errorResponse})
+	//nolint:staticcheck // TODO: FIX SA1019: gnmi.Error is deprecated: Do not use.
```

Comment:
I would not silence this as it is a **real** todo that needs fixing.

Reply:
Reverted.